scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv_helper_test.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import preprocess_csv_helper
+
+class PreprocessCSVHelperTest(unittest.TestCase):
```

Comment:
Can you also add tests for the core (non helper function) preprocess_csv.py? May need to modularize the code a bit for easier testing (1 - get data into a data frame; 2 - act on the data frame). I suggest having a testdata subdir where you put a few edge case rows of the data, and read in that dataframe to test the second part.